src/blocks/slackBlocks.ts

```diff
@@ -13,9 +21,23 @@ export function markdown(text: string): MrkdwnElement {
   };
 }
 
+export function plaintext(text: string): PlainTextElement {
+  return {
+    type: 'plain_text',
+    text,
+  };
+}
+
 export function section(block: MrkdwnElement): KnownBlock {
   return {
     type: 'section',
     text: block,
   };
 }
+
+export function SectionBlock(fields: MrkdwnElement[]): KnownBlock {
```

Comment:
Would be good to add this to the above `section()` function.

`export function section(block: MrkdwnElement, fields: MrkdownElement[])`

Reply:
since section is already being used by other functions rewriting it is probably out of scope, spoke with Hubert and trying to make either parameter optional just makes the code too messy and unreadable, so I'm going to keep it as two functions